docs/hardware/4-nrf91/2-quickstart/2-set-up-zephyr.md

```diff
@@ -35,7 +35,7 @@ Your system is all set up and ready to start building & flashing with Zephyr. Ve
 
 ```
 cd ~/zephyr-nrf/zephyr
-west build -p auto -b  circuitdojo_feather_nrf9160_ns samples/basic/minimal
+west build -p auto -b  circuitdojo_feather_nrf9160ns samples/basic/minimal
```

Comment:
This is wrong. There should be underscore. At least with newest SDK (`main` branch).

Reply:
You are correct. The current state of this PR keeps the SDK branch at 1.7.0-rc2 which still has the old name without the underscore.

I'd like to get this huge PR closed so that the toolchain directions are live as soon as possible. Let's leave this change in order to finish here, and then I will work on the PR #139 you submitted that updates to 1.7.1. I'll test it locally to confirm, then make sure your PR can still be merged after closing this one.

Reply:
Then consider adding https://github.com/golioth/docs/pull/139#discussion_r779080454 change (with older revision of SDK, which still supported 1.7.0-rc2) to this PR, as 1.7.0-rc2 is not compatible with `main` branch of SDK.